### ISSUE TITLE
Zoom levels & fix aspect ratio needed on all slide elements

### ISSUE THREAD

**Issue by nuke-web3** [Polkadot-Blockchain-Academy/pba-content#135]
see the correct behavior of slide contents at this commit: 
`git checkout 65657db9d51f630f4f07f22d4fc94f717184dce6`
https://github.com/paritytech/polkadot-blockchain-academy/commit/65657db9d51f630f4f07f22d4fc94f717184dce6

where zoom level only changes the watermark behavior.

> Did you test zoom levels and mobile? All slides must have the same relative size on all screens. This should not be reactive.

_Originally posted by @NukeManDan in https://github.com/paritytech/polkadot-blockchain-academy/pull/124#discussion_r905095570_

See the comments in #124 for screenshots of problematic behavior.

### Goal: all devices see the same relative size of slides rendered 
(as you can test in 65657db9d51f630f4f07f22d4fc94f717184dce6)

In other words I think are correct: the slides _should not_ be reactive so-to-speak. They should be static, resized to fix the window they exist in to have the same relative sizes & positions.
